### ISSUE TITLE
move title and back-arrow of a subview (iOS)

### ISSUE THREAD

**Issue by mase60** [Nerwyn/material-rounded-theme#7]
**Is your feature request related to a problem? Please describe.**
position of the backarrow and title of a subview is off

**Describe the solution you'd like**
adding following code moves both items:

```
      .main-title {
        padding-bottom: 10px;
      }
      .mdc-icon-button {
        padding-bottom: 20px;
      }
      ha-icon-button-arrow-prev {
        padding-bottom: 10px;
        padding-left: 20px;
      }
```

**Additional context**

how it looks right now (looks way worse on an iphone 13 pro)

![photo_2024-03-01_15-34-01](https://github.com/Nerwyn/material-rounded-theme/assets/5972205/b24f7e9f-cf7d-4186-8824-4f8d0f80698f)

how it looks fixed:

![photo_2024-03-01_15-34-10](https://github.com/Nerwyn/material-rounded-theme/assets/5972205/54646993-b9c6-4e11-ad8f-0f6a046b7364)

**Comment by Nerwyn** [comment]
Adding some left padding/margin to the subview back button makes sense, but I'm not sure about the other changes or if this is the best way to go about it since these changes would affect the current view edit mode buttons. `ha-icon-button-arrow-prev` is also used for the edit mode move view left button so that selector is too general. Do the icon and label need to be shifted up? As they are now they're centered and look fine on my testing iPhone SE.

**Comment by Nerwyn** [comment]
Narrowing the back button selector using the slot field fixes that issue.

```css
/* Move subview back button to the right */
ha-icon-button-arrow-prev[slot="navigationIcon"] {
  padding-left: 20px;
}
```

**Comment by Nerwyn** [comment]
Back button fix is in [1.5.2-beta.001](https://github.com/Nerwyn/material-rounded-theme/releases/tag/1.5.2-beta.001)

**Comment by mase60** [comment]
> Adding some left padding/margin to the subview back button makes sense, but I'm not sure about the other changes or if this is the best way to go about it since these changes would affect the current view edit mode buttons. `ha-icon-button-arrow-prev` is also used for the edit mode move view left button so that selector is too general. Do the icon and label need to be shifted up? As they are now they're centered and look fine on my testing iPhone SE.

didn´t know that the changes also effect the edit mode buttons, since i don´t use them in my theme.
in my opinion they seem to be to far to the bottom, at least on my 13pro. but that´s maybe personal preference.

will give your beta a try :)

**Comment by Nerwyn** [comment]
@mase60 if there's no other feedback I plan on releasing this fix later today.

**Comment by mase60** [comment]
sorry, didn‘t had the time to check it yet but what you said makes sense. so i guess it‘s good to release that fix.

**Comment by Nerwyn** [comment]
Fixed in [1.5.2](https://github.com/Nerwyn/material-rounded-theme/releases/tag/1.5.2).
